### PR TITLE
app README, closes #13

### LINKED ISSUE #13: Doc: How to INSTALL
For 0.1 or 0.2, it would be great to have a HOWTO or INSTALL file or section in the README.

Something like a QUICKSTART would also be great.

### PR DESCRIPTION
**Pull request by stefan2904**
(first iteration, please dont merge, please add feedback.)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,13 +2,15 @@
 
 [![Build Status](https://travis-ci.org/whawty/auth.svg?branch=master)](https://travis-ci.org/whawty/auth)
 
-whawty.auth is a simple file based authentication suite. It uses flat files containing password hashes. These
+whawty.auth is a simple file based authentication suite. The **store** uses flat files containing password hashes. These
```
Comment (equinox0815) [inline README.md]: s/The/It's/

File: app/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+The easiest way to install the *whawty.auth app* is by installing one of the packages. For example, Debian packages can be found in the [spreadspace APT Repository](https://build.spreadspace.org/).
+
+
+### go get
+
+You can install & build the latest development version using `go get`:
+
+```
+$ mkdir whawty-auth
```
Comment (equinox0815) [inline app/README.md:15]: $ cd whawty-auth

**Review by equinox0815 — COMMENTED** [review]
Looks good so far

File: README.md (hunk 1 above)
Comment (stefan2904) [reply README.md]: fixed.

File: app/README.md (hunk 2 above)
Comment (stefan2904) [reply app/README.md:15]: oh, thanks for that. fixed.

File: README.md [hunk 3]
```diff
@@ -2,13 +2,15 @@
 
 [![Build Status](https://travis-ci.org/whawty/auth.svg?branch=master)](https://travis-ci.org/whawty/auth)
 
-whawty.auth is a simple file based authentication suite. It uses flat files containing password hashes. These
+whawty.auth is a simple file based authentication suite. It's **store** uses flat files containing password hashes. These
```
Comment (nicoonoclaste) [inline README.md]: “Its”, not “It's”

File: README.md [hunk 4]
```diff
@@ -2,13 +2,15 @@
...
 hashes are currently based on scrypt but the algorithm can be upgraded to newer/stronger formats on the fly.
 To find out more about the storage backend read [this file](doc/SCHEMA.md).
 
-This repository contains a golang app which can act as a whawty.auth agent. You can use this app to manage
-usernames using a simple web UI. It also offers a _saslauthd_ compatible unix socket to authenticate against it.
-This socket is also used by the PAM module which can be used to bring whawty.auth to PAM applications.
+This repository contains a golang **app** which can act as a whawty.auth agent. You can use this app to manage
+usernames using a simple web UI.
+Install instructions can be found [here](app/README.md)
```
Comment (nicoonoclaste) [inline README.md]: Missing period.

File: README.md [hunk 5]
```diff
@@ -2,13 +2,15 @@
...
-whawty.auth is a simple file based authentication suite. It uses flat files containing password hashes. These
+whawty.auth is a simple file based authentication suite. It's **store** uses flat files containing password hashes. These
 hashes are currently based on scrypt but the algorithm can be upgraded to newer/stronger formats on the fly.
 To find out more about the storage backend read [this file](doc/SCHEMA.md).
 
-This repository contains a golang app which can act as a whawty.auth agent. You can use this app to manage
-usernames using a simple web UI. It also offers a _saslauthd_ compatible unix socket to authenticate against it.
-This socket is also used by the PAM module which can be used to bring whawty.auth to PAM applications.
+This repository contains a golang **app** which can act as a whawty.auth agent. You can use this app to manage
+usernames using a simple web UI.
```
Comment (nicoonoclaste) [inline README.md]: “manage users”, not “manage usernames”

**Review by nicoonoclaste — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (stefan2904) [reply README.md]: fixed

File: README.md (hunk 3 above)
Comment (stefan2904) [reply README.md]: fixed

File: README.md (hunk 4 above)
Comment (stefan2904) [reply README.md]: fixed

**Review by nicoonoclaste — APPROVED** [review]

**Pull request merged**